yt_dlp/extractor/boosty.py

```diff
@@ -0,0 +1,92 @@
+from yt_dlp.utils.traversal import traverse_obj
+from .common import InfoExtractor
+from ..utils import qualities, ExtractorError
+
+
+class BoostyIE(InfoExtractor):
+    _VALID_URL = r"https?://boosty.to(/[\w-]+/posts/)(?P<id>[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})"
+    _TESTS = [{
+        "url": "https://boosty.to/ikakprosto/posts/595db00d-7906-4d0e-99d6-1067d79ecefe",
+        "info_dict": {
+            "id": "595db00d-7906-4d0e-99d6-1067d79ecefe",
+            "ext": "mp4",
+            "title": "КАК YOUTUBE и TWITTER СВОДЯТ РОССИЯН С УМА - [когнитивные воины 2]",
+            "thumbnail": r"re:^https://i.mycdn.me/videoPreview.*",
+            "author": "ikakprosto",
+            "duration": 11186,
+            "tags": [],
+            "alt_title": "Уралов.mp4",
+            "display_id": "https://ok.ru/videoembed/4358687697659",
+            "channel": "ikakprosto",
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        json_data = self._search_json(
+            r'<script type="text/plain" id="initial-state">',
+            webpage, "boosty", video_id,
+        )
+        data = json_data["posts"]["postsList"]["data"]["posts"][0]
+        for i in data["data"]:
```

Comment:
Are there any posts with multiple videos?

Reply:
Here is a post with multiple (3)  videos: https://boosty.to/maddyson/posts/0c652798-3b35-471f-8b48-a76a0b28736f

Reply:
Thanks for the info and example link. In that case, the extractor should return multiple videos in a playlist and not be hardcoded for the first video.
This PR is stale, but if you (or anyone else wants) they can open a new PR to implement the extractor.